.ci/pipeline_definitions

```diff
@@ -1,24 +1,12 @@
 gardener-website-generator:
-  template: 'default'
   base_definition:
-    traits:
-      version: ~
-      scheduling:
-        suppress_parallel_execution: true
```

Comment:
this is actually a change in behaviour as this attributes defaults to `false`, it controls ["whether parallel executions of the same job should be prevented"](https://gardener.github.io/cc-utils/traits/scheduling.html)

Reply:
you have to decide whether you want to allows this behaviour, or not :-)

Reply:
when could parallel execution of the same job happen?

Reply:
for both manual and automated triggers

let's say you have build trigger on repo head update.
if two head updates happen in a short period of time, two head update jobs could be scheduled and executed in parallel.